.npmrc

```diff
@@ -1,2 +1,2 @@
-//npm.pkg.github.com/:_authToken=${NPM_TOKEN}
```

Comment:
Is this unintentional or why has the NPM_TOKEN reference changed?

Reply:
I was wrong - it did matter